focus_areas/community_growth.md

```diff
@@ -1,13 +1,13 @@
 ## Community Growth
 
-Goal: Identify the size of the project community and whether it's growing, shrinking, or staying the same.
+Goal: Identify the size of the project community and whether it is growing, shrinking, or staying the same.
```

Comment:
I'm pretty sure there's no difference between "it's" and "it is".

Reply:
So no harm done

Reply:
Sorry, I have mistaken. I thought it was `its`.